asciidoc-markup-sample-doc/asciidoc-markup-samples.adoc

```diff
@@ -4,7 +4,7 @@
 
 = AsciiDoc Mark-up Quick Reference for Red{nbsp}Hat Documentation
 
-== Overview of AsciiDoc Markup Elements used in Red{nbsp}Hat Documentation
+== AsciiDoc Markup Elements used in Red{nbsp}Hat Documentation
```

Comment:
IBM Styleguide calls for sentence case on titles.


```suggestion
== AsciiDoc markup elements used in Red{nbsp}Hat documentation
```

Reply:
Sure thing. I'll fix the "Additional Resources" section at the bottom too.

The title of the doc should probably also be consistent. Any objections to also updating that? I'm sure that existing documents that link here will have it title case, but I don't think that's a big deal to have that mismatch.

Reply:
I'm okay with another PR that makes all titles 'right'.  This is just something that always jumps out at me because it's on our peer review checklist :)